### ISSUE TITLE
Helpers seem to be shuffled.

### ISSUE THREAD

**Issue by hourliert** [cazala/mnist#1]
Hi,

I'm trying to use your library but it seems that when I try to get a digit for 0, 2, 3, 4, 6, 7, 8, 9 I always get a wrong digit.
It seems that the mapping of real digits is wrong.

I always get this mapping:

| desired digit | real digit |
| --- | --- |
| 0 | 8 |
| 1 | 1 |
| 2 | 4 |
| 3 | 6 |
| 4 | 6 |
| 5 | 5 |
| 6 | 9 |
| 7 | 3 |
| 8 | 7 |
| 9 | 0 |

Here the code that generates the following image:

``` javascript
  for (var i = 0; i < 10; i++) {
    var digit = mnist[i].get();

    var el: HTMLCanvasElement = <HTMLCanvasElement>document.getElementById('canvas' + i);

    var context = el.getContext('2d');

    context.fillStyle = 'white';
    context.fillRect(0, 0, el.width, el.height);

    console.info('Drawing...');
    mnist.draw(digit, context, 0, 0);

    // (<any>context).imageSmoothingEnabled = true;
    context.drawImage( el, 0, 0, 4*el.width, 4*el.height );
  } 
```

<img width="715" alt="screen shot 2015-08-24 at 6 12 32 pm" src="https://cloud.githubusercontent.com/assets/7560406/9445305/bf79857a-4a8b-11e5-818b-560006f37690.png">

Did I miss something ?

Btw, I am using JSPM with typescript.

**Comment by NoPinky** [comment]
I also have this issue.  And you also notice, there is no way to get a 2 while the dataset for 6 is doubled. That is a strangely grave mistake and I couldn't imagine how this could happen programmatically.
The faulty data lies within the /src/*.json files. But the library uses a packed mnist.js file, where the json's are already included.

**Comment by cazala** [comment]
This is fixed in [1.0.3](https://github.com/cazala/mnist/releases/tag/1.0.3)
